src/Components/CheckboxGroup/BasicCheckboxGroup.js

```diff
@@ -27,13 +27,25 @@ const BasicCheckboxGroup = ({ stateVariable, options }) => {
     const optionKeys = Object.keys(options);
 
     const formControlLabels = optionKeys.map((optionKey) => {
+      const splitLabel = options[optionKey].props.defaultMessage.split(':');
```

Comment:
Does this work for Spanish? You might have to split them into 2 separate messages in the config.